templates/search/search_clients_last_seen_v1/init.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE
+  search_clients_last_seen_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id,
+  client_id
+OPTIONS
+  ( require_partition_filter=TRUE) AS
+SELECT
+    CAST(NULL AS DATE) AS submission_date,
```

Comment:
Explicitly defined the column ordering here (rather than `SELECT *`) to match what we do in `_grouped` in `query.sql`.